README.md

```diff
@@ -8,16 +8,30 @@
 
 ---
 
-**ProbNum implements probabilistic numerical methods in Python.** Such methods solve numerical problems from linear
-algebra, optimization, quadrature and differential equations using _probabilistic inference_. This approach captures 
-uncertainty arising from _finite computational resources_ and _stochastic input_. 
+ProbNum is a Python toolkit for solving numerical problems in linear algebra, optimization, quadrature and
+differential equations. ProbNum solvers not only estimate the solution of the numerical problem, but also its
+uncertainty (the error) which certainly arises from finite computational resources, discretization, and stochastic input.
```

Comment:
I am missing the point about numerical approximation being framed as inference and a clearer pointer to probabilistic numerics to be honest.

Reply:
I appended the following sentence as it illustrates a use case of the ProbNum library: "These uncertainties can be used in downstream decisions."

The pointer to the research field "probabilistic numerics" and what it implies is just below this paragraph.